packages/hooks/src/useTotalAccountBalance.ts

```diff
@@ -1,22 +1,20 @@
 import useRealTotalAssetCurrency from './useTotalAssetCurrency';
 import useExchangeRate from './useExchangeRate';
-
 /**
  * we can use this hook to get the total balance of the given accounts list.
  * it loops through the accounts list and adds the balance of each account
  * to the total balance, it also converts the balance to the currency of the
  * first account in the list
  */
-
-const useTotalAccountBalance = (accounts: { balance?: number; currency?: string }[]) => {
+const useTotalAccountBalance = (accounts: { balance?: number; currency?: string; account_type?: string }[]) => {
```

Comment:
```suggestion
type TAccounts = {
    balance?: number;
    currency?: string;
    account_type?: string
};

const useTotalAccountBalance = (accounts: TAccounts[]) => {
```
@shontzu-deriv Please extract it into a type above the function 🙏🏻